src/jdk.jlink/share/classes/jdk/tools/jlink/internal/ImageFileCreator.java

```diff
@@ -38,6 +38,7 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
+import java.util.Comparator;
```

Comment:
nit: `Comparator` might no longer be needed? I don't see where it's used.

Reply:
Yep. Looks like it's a leftover. Thanks!